app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -483,7 +483,7 @@ class BrowserFragment :
         } else if (sessionFeature.get()?.onBackPressed() == true) {
             return true
         } else {
-            if (tab.source == SessionState.Source.ACTION_VIEW || tab.isCustomTab()) {
+            if (tab.source is SessionState.Source.External.ActionView || tab.isCustomTab()) {
```

Comment:
In Fenix I've changed a similar check to just be `tab.source is SessionState.Source.External`. This covers not just ActionView but also other intents we could be opened with (ActionSend, ActionSearch), and a CustomTab source (assuming it was set correctly). You'd want to verify that `tab.isCustomTab()` will produce the same result as `tab.source is SessionState.Source.External.CustomTab`.

Reply:
Changed this to `tab.source is SessionState.Source.External` as recommended. I think `tab.isCustomTab()` is fine as-is since it checks `SessionState` is an instance of `CustomTabSessionState`.